spec/decorators/ncr_work_order_decorator_spec.rb

```diff
@@ -18,4 +18,17 @@
     wo = create(:ncr_work_order, :with_approvers).decorate
     expect(wo.current_approver_email_address).to eq(wo.approvers.first.email_address)
   end
+
+  it "returns current approver email address when status is pending" do
+    wo = create(:ncr_work_order, :with_approvers).decorate
+    expect(wo.pending?).to be_truthy
+    expect(wo.status_aware_approver_email_address).to eq(wo.current_approver_email_address)
+  end
+
+  it "returns final approver when status is approved" do
+    wo = create(:ncr_work_order, :with_approvers).decorate
+    wo.proposal.approve!
+    expect(wo.approved?).to be_truthy
```

Comment:
Don't think this needs to change now, but for future I would say let's avoid `truthy` in favor or RSpec predicate matchers

`expect(wo).to be_approved`

Super long convo on this topic and explanations for why here https://github.com/thoughtbot/guides/pull/191